component/src/test/java/com/synopsys/integration/alert/component/settings/proxy/actions/SettingsProxyCrudActionsTest.java

```diff
@@ -0,0 +1,212 @@
+import com.synopsys.integration.alert.component.settings.descriptor.SettingsDescriptorKey;
+import com.synopsys.integration.alert.component.settings.proxy.action.SettingsProxyCrudActions;
+import com.synopsys.integration.alert.component.settings.proxy.database.accessor.SettingsProxyConfigAccessor;
+import com.synopsys.integration.alert.component.settings.proxy.model.SettingsProxyModel;
+import com.synopsys.integration.alert.component.settings.proxy.validator.SettingsProxyValidator;
+import com.synopsys.integration.alert.database.settings.proxy.NonProxyHostsConfigurationRepository;
+import com.synopsys.integration.alert.database.settings.proxy.SettingsProxyConfigurationEntity;
+import com.synopsys.integration.alert.database.settings.proxy.SettingsProxyConfigurationRepository;
+import com.synopsys.integration.alert.descriptor.api.model.DescriptorKey;
+import com.synopsys.integration.alert.test.common.AuthenticationTestUtils;
+import com.synopsys.integration.alert.test.common.MockAlertProperties;
+
+public class SettingsProxyCrudActionsTest {
+    private static final String HOST = "hostname";
+    private static final Integer PORT = 12345;
+    private static final String USERNAME = "userName";
+    private static final String PASSWORD = "password";
+
+    private final Gson gson = new Gson();
+    private final AlertProperties alertProperties = new MockAlertProperties();
+    private final FilePersistenceUtil filePersistenceUtil = new FilePersistenceUtil(alertProperties, gson);
+    private final EncryptionUtility encryptionUtility = new EncryptionUtility(alertProperties, filePersistenceUtil);
+
+    private final AuthenticationTestUtils authenticationTestUtils = new AuthenticationTestUtils();
+    private final DescriptorKey descriptorKey = new SettingsDescriptorKey();
+    private final PermissionKey permissionKey = new PermissionKey(ConfigContextEnum.GLOBAL.name(), descriptorKey.getUniversalKey());
+    private final Map<PermissionKey, Integer> permissions = Map.of(permissionKey, 255);
+    private final AuthorizationManager authorizationManager = authenticationTestUtils.createAuthorizationManagerWithCurrentUserSet("admin", "admin", () -> new PermissionMatrixModel(permissions));
+
+    private final SettingsProxyValidator settingsProxyValidator = new SettingsProxyValidator();
+    private final SettingsDescriptorKey settingsDescriptorKey = new SettingsDescriptorKey();
+
+    @Test
+    public void getOneTest() {
+        UUID uuid = UUID.randomUUID();
+        SettingsProxyConfigurationRepository settingsProxyConfigurationRepository = Mockito.mock(SettingsProxyConfigurationRepository.class);
+        NonProxyHostsConfigurationRepository nonProxyHostsConfigurationRepository = Mockito.mock(NonProxyHostsConfigurationRepository.class);
+
+        Mockito.when(settingsProxyConfigurationRepository.findById(Mockito.any())).thenReturn(Optional.of(createSettingsProxyConfigurationEntity(uuid)));
```

Comment:
findById(Mockito.any()) you could change it to Mockito.eq(uuid) so that it matches exactly.  This forces Mockito to only create the entity for the matching UUID which you are testing.  I've been getting into the habit of using Mockito.eq to perform an exact match on the arguments rather than any().